ebu_tt_live/ui/assets/css/main.css

```diff
@@ -10,6 +10,57 @@
 
 }
 
+body {
+    padding-top: 5px;
+}
+
+#header {
+    padding: 10px;
+    background: rgb(255,255,255); /* Old browsers */
+    background: -moz-linear-gradient(top, rgba(255,255,255,1) 0%, rgba(237,238,244,1) 6%, rgba(210,218,224,1) 49%, rgba(242,244,247,1) 96%, rgba(255,255,255,1) 100%); /* FF3.6-15 */
+    background: -webkit-linear-gradient(top, rgba(255,255,255,1) 0%,rgba(237,238,244,1) 6%,rgba(210,218,224,1) 49%,rgba(242,244,247,1) 96%,rgba(255,255,255,1) 100%); /* Chrome10-25,Safari5.1-6 */
+    background: linear-gradient(to bottom, rgba(255,255,255,1) 0%,rgba(237,238,244,1) 6%,rgba(210,218,224,1) 49%,rgba(242,244,247,1) 96%,rgba(255,255,255,1) 100%); /* W3C, IE10+, FF16+, Chrome26+, Opera12+, Safari7+ */
```

Comment:
that is no box shadow. That is a gradient as well to make them look like a button which essentially they are to prompt the intuitive ux user to click them. Shadow is behind the header panel which I removed as well now as well as its gradient background